tests/snippet_tests/test_query_points.rs

```diff
@@ -56,6 +60,62 @@ async fn test_query_points() {
                 .using("colbert")
                 .limit(10u64)
         ).await?;
+        
+        // Random sampling (as of 1.11.0)
+        let _sampled = client
+            .query(
+                QueryPointsBuilder::new("{collection_name}")
+                    .query(Query::new_sample(Sample::Random))
+            )
+            .await?;
+        
+        // Score boost depending on payload conditions (as of 1.14.0)
+        let _tag_boosted = client.query(
+            QueryPointsBuilder::new("{collection_name}")
+                .add_prefetch(PrefetchQueryBuilder::default()
+                    .query(vec![0.01, 0.45, 0.67])
+                    .limit(100u64)
+                )
+                .query(FormulaBuilder::new(Expression::sum_with([
+                    Expression::variable("$score"),
```

Comment:
I think it makes sense to make a dedicated function for $score, what do you think?

Maybe just `Expression::score()`